packages/shared/src/components/plus/PlusMarketingModal.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import dynamic from 'next/dynamic';
+import type { ModalProps } from '../modals/common/Modal';
+import { Modal } from '../modals/common/Modal';
+import { checkIsExtension } from '../../lib/func';
+import { useBoot, useViewSize, ViewSize } from '../../hooks';
+import { ModalClose } from '../modals/common/ModalClose';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { PaymentContextProvider } from '../../contexts/PaymentContext';
+import { MarketingCtaVariant } from '../marketingCta/common';
+import { useLogContext } from '../../contexts/LogContext';
+import { LogEvent, TargetType } from '../../lib/log';
+import { ModalSize } from '../modals/common/types';
+
+const PlusMobileDrawer = dynamic(
+  () => import(/* webpackChunkName: "plusMobileDrawer" */ './PlusMobileDrawer'),
+);
+
+const PlusExtension = dynamic(
+  () => import(/* webpackChunkName: "plusExtension" */ './PlusExtension'),
+);
+
+const PlusWebapp = dynamic(
+  () => import(/* webpackChunkName: "plusWebapp" */ './PlusWebapp'),
+);
+
+const PlusMarketingModal = (modalProps: ModalProps): ReactElement => {
+  const { getMarketingCta, clearMarketingCta } = useBoot();
+  const marketingCta = getMarketingCta(MarketingCtaVariant.Plus);
+  const { campaignId } = marketingCta;
+  const { logEvent } = useLogContext();
+  const { closeModal } = useLazyModal();
+  const isExtension = checkIsExtension();
+  const isMobile = useViewSize(ViewSize.MobileL);
```

Comment:
We have different mobile sizes, like `MobileL` and `MobileM`. I think it is best to check whether it is Tablet and up.

```suggestion
  const isTablet = useViewSize(ViewSize.Tablet);
```